source/api/app.py

```diff
@@ -411,31 +411,30 @@ def get_data_columns():
         keys = json.loads(app.current_request.raw_body.decode())["s3key"]
         keys_to_validate = [x.strip() for x in keys.split(",")]
 
-        # for concurrent glue jobs, can only run a max of 200 per account
-        if len(keys_to_validate) > 200:
-            return Response(
-                body={"message": "Number of files selected cannot exceed 200"},
-                status_code=400,
-                headers={"Content-Type": "text/plain"},
-            )
-
-        # get first columns to compare against to ensure all files have same schema
-        base_key = keys_to_validate[0]
-
         json_content_type = "application/json"
         csv_content_type = "text/csv"
+        plain_text_content_type = "text/plain"
         content_type = ""
+
+        # for concurrent glue jobs, can only run a max of 200 per account
+        if(len(keys_to_validate) > 200):
+            return Response(body={"message": "Number of files selected cannot exceed 200."},
```

Comment:
Should the `200` be an environment variable, instead of hardcoding? It can still fallback to 200, if its not set in the env.
```suggestion
       max_concurrent_glue_job = os.environ.get("MAX_CONCURRENT_GLUE_JOB", 200)
        if(len(keys_to_validate) > max_concurrent_glue_job):
            return Response(body={"message": f"Number of files selected cannot exceed {max_concurrent_glue_job}."},
```

Reply:
It's probably passing unit test, but might have to update unit test, to test max concurrency [here](https://github.com/aws-solutions/amazon-marketing-cloud-uploader-from-aws/blob/development/source/api/tests/test_api_unit_test.py#L102). Just incase this logic is removed in the future, the unit test will fail.

Reply:
You can request 200 to be user configurable by submitting a feature request under github issues, but it's unlikely that anyone would use that feature because of the default quotas for Glue, see: https://docs.aws.amazon.com/general/latest/gr/glue.html